src/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import { Provider } from 'react-redux';
-import App from './App';
+import App from './components/App/App';

```

Comment:
This import path is a bit ugly - could be maybe move to `components/App/index.tsx` and import from `./components/App`? (I _think_ that works, check it though)

Reply:
@bencpeters 

I am facing an error with that 😶

Reply:
Is it similar to [this issue](https://github.com/Microsoft/vscode/issues/45289)?

It looks like the default export of the `index.tsx` file might need to match the name of the module (rather than being wrapped in a HOC which `withStyles` does.

If so, you could try something like:

```javascript
function AppComponent(...) {
  ...
}

const App = withStyles(AppComponent);
export default App;
```

Does that work?

Reply:
This might depend on the babel config with TypeScript too